### DISCUSSION TITLE
Small typo on supabase.io/auth

### DISCUSSION THREAD

**Discussion by oguzhanmeteozturk** [supabase/supabase#1259] [Questions] [answered]
Under Community driven examples, libraries and guides Hiro1107's solution is advertised as a Next.js example while it's a Nest.js example

**Comment by silentworks** [comment, marked as answer]
Hi @ACAVJW4H thanks for pointing this out, if you have time and want to contribute to this change you can send over a PR to the repo with the change. You can find the file to change here https://github.com/supabase/supabase/blob/master/web/docs/guides/examples.mdx

**Comment by oguzhanmeteozturk** [comment]
Hey silentworks I think the typo was on the main website rather than docs. I submitted a PR #1289

**Reply by silentworks** [reply]
Nice one.
